File: components_page/components/card/color.R
library(dashBootstrapComponents)

card_content <- list(
  dbcCardHeader("Card header"),
  dbcCardBody(
    list(
      html$h5("Card title", className = "card-title"),
      p(
        "This is some card content that we'll reuse",
        className = "card-text"
      )
    )
  )
)

cards <- div(
  list(
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "primary", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "secondary", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "info", inverse = TRUE))
      ),
      className = "mb-4"
    ),
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "success", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "warning", inverse = TRUE)),
        dbcCol(dbcCard(card_content, color = "danger", inverse = TRUE))
      ),
      className = "mb-4"
    ),
    dbcRow(
      list(
        dbcCol(dbcCard(card_content, color = "light")),
        dbcCol(dbcCard(card_content, color = "dark", inverse = TRUE))
      )
    )
  )
)

File: docs/components_page/components/placeholder/color.py
import dash_bootstrap_components as dbc
from dash import html

placeholder = html.Div(
    [
        dbc.Placeholder(color="primary", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="secondary", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="success", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="warning", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="danger", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="info", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="light", className="me-1 mt-1 w-100"),
        dbc.Placeholder(color="dark", className="me-1 mt-1 w-100"),
    ]
)
